CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+
+## [v18.0.0] 2023-08-28
```

Comment:
I think we're missing updates at the bottom of the CL to link to this version

Reply:
Yeah, is it possible that you created `release-18.0.0` from `master` as the base instead of `release-18.0.x`? It's not critical, just trying to figure out what happened.